src/p_saveg.c

```diff
@@ -1725,12 +1725,12 @@ static void saveg_read_scroll_t(scroll_t *str)
     // fixed_t last_height;
     str->last_height = saveg_read32();
 
-    // fixed_t vdy;
-    str->vdy = saveg_read32();
-
     // fixed_t vdx;
     str->vdx = saveg_read32();
 
+    // fixed_t vdy;
+    str->vdy = saveg_read32();
+
```

Comment:
This bug may break some saves. Try to save at the beginning of MAP02 AD_MORTEM.wad, enemies don't spawn later. Still compatible, no crashes.

Reply:
I can live with that, no reason for a savegame version bump.